flocker/provision/_install.py

```diff
@@ -104,6 +102,20 @@ def get_repository_url(distribution, flocker_version):
         raise UnsupportedDistribution()
 
 
+def get_repo_options(flocker_version):
+    """
+    Get a list of options for enabling necessary yum repositories.
+
+    :param bytes flocker_version: The version of Flocker to get options for.
+    :return: List of bytes for enabling (or not) a testing repository.
+    """
+    is_dev = not is_release(flocker_version)
```

Comment:
> How can I simulate both a dev and a release version locally?

Do a dev release and a marketing release :smile: 

I'm hoping that the tests are sufficient.